binaries/cli/src/main.rs

```diff
@@ -368,11 +368,13 @@ fn run() -> eyre::Result<()> {
                 &mut *session,
             )?;
 
+            println!("Started dataflow {dataflow_id}");
```

Comment:
The idea behind having only the dataflow id was to make it programmable to store the ID as a variable or within a file.

docker run or start gives you the name or ID as a return as well so that is why I pushed for this version.

Reply:
Ah, I remember. Maybe we add some flag to get just the UUID as output? Then users that want to use `dora start --detach` in scripts still have a way to do that.

Reply:
Honestly it seems that the CI/CD is breaking. I'm not sure if that is a problem we want to work on now, as I rarely get feedback on how this is confusing.

I also do a lot of copy pasting of IDs and it would be harder with this.

Reply:
Sure, we can leave it as is if you prefer. I just thought that also printing the dataflow _name_ would be useful so that you don't need to use the long UUID.

(The CI failures are easy to fix, I just forgot to update the code for the `multiple-daemons` example.)